lib/script/script.js

```diff
@@ -106,11 +106,11 @@ Script.fromBuffer = function(buffer) {
 };
 
 Script.prototype.toBuffer = function() {
```

Comment:
Do we want to add a doc description also for all the methods inside this class?

Reply:
Yes I think we should, but doing it now may not be the best moment? Besides there are some functions like this one that may not need any doc? 
I think right now the priority is to get things working for zendoo. Then once that is done, we can come back to the project and update many things: node dependencies, js code, and comments  😄